src/lib/plugin.js

```diff
@@ -158,13 +165,20 @@ class FiveBellsLedger extends EventEmitter2 {
     }
 
     const accountUri = this.account
+    const accountProtocol = parseURL(accountUri).protocol
+    if (accountProtocol !== 'http:' && accountProtocol !== 'https:') {
+      throw new Error('Invalid account URI')
+    }
 
     // Resolve account information
-    const res = yield requestRetry({
+    const res = yield this._requestRetry({
       method: 'GET',
       uri: accountUri,
       json: true
-    }, 'Failed to resolve ledger URI from account URI', this.credentials)
+    }, {
+      errorMessage: 'Failed to resolve ledger URI from account URI',
```

Comment:
This is still a pretty cryptic error message to get if the account simply doesn't exist. If GETting the account returns a 404 I would expect a message like "cannot connect to account" or "account does not exist"